docs/sdks/python/GETTING_STARTED.md

```diff
@@ -23,7 +23,7 @@ Once your SDK object is set, create any of the Appwrite service objects and choo
 ```python
 users = Users(client)
 
-result = users.create('[USER_ID]', 'email@example.com', 'password')
+result = users.create(ID.unique(), email = 'email1@example.com', phone = '+123456789', password = 'password', name = 'user1')
```

Comment:
Why `email1`?

Reply:
Actually, that I used for demonstrating an example above.
So shall I rename it to `email` or it's fine?